packages/sanity/src/desk/comments/src/components/avatars/SpacerAvatar.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
+import {studioTheme} from '@sanity/ui'
 
-export const AVATAR_HEIGHT = 25
+export const AVATAR_HEIGHT = studioTheme.avatar.sizes[0].size
```

Comment:
This value should be `25` (the same width as the avatars used in comments) for the spacing to be correct in the UI. The values from `studioTheme` for avatar sizes do not have 25, so this value must be hardcoded here to 25.

<img width="765" alt="Screenshot 2023-12-19 at 08 30 21" src="https://github.com/sanity-io/sanity/assets/15094168/8e602c79-7602-4d3f-bd40-2e705125afed">